packages/ui/Slottable/Slottable.js

```diff
@@ -60,7 +60,7 @@ const Slottable = hoc(defaultConfig, (config, Wrapped) => {
 		slots.forEach(slot => {
 			const dist = distributed[slot];
 			const prop = props[slot];
-			if (prop && dist) {
+			if (prop != null && dist != null) {
 				distributed[slot] = [].concat(prop, dist);
 			} else if (prop) {
```

Comment:
I think this one should be `!= null`, too.

Reply:
Yeah, i agree.